### DISCUSSION TITLE
What does link-workspace-packages deep do?

### DISCUSSION THREAD

**Discussion by gajus** [pnpm/pnpm#7221] [Q&A]
> If this is enabled, locally available packages are linked to node_modules instead of being downloaded from the registry. This is very convenient in a monorepo. If you need local packages to also be linked to subdependencies, you can use the deep setting.

https://pnpm.io/npmrc#link-workspace-packages

It is mentioned in documentation without any explanation of what it does.

**Comment by zkochan** [comment]
It is right there: If you need local packages to also be linked to subdependencies, you can use the `deep` setting. When it is true, only the direct deps are replaced with workspace packages.
